### ISSUE TITLE
Feature request: customised input styles

### ISSUE THREAD

**Issue by whiskeyalfaone** [AnySoftKeyboard/AnySoftKeyboard#1890]
I have both French and English keyboards.

I like to use my keyboards separately (i.e French is one keyboard, English is another), however I want to be able to use the AZERTY layout for both languages.

I find that there is nothing more infuriating than the layout changing for the language, especially when typing my login password, because that is learnt to muscle memory (try changing to a different layout and see how much longer it takes you to type in a password that you use all the time).

In the AOSP keyboard I am able to add a 'customised input style' for Engish, using the AZERTY layout and then enable it from the list of keyboards.

I know I can do the language override by holding enter, however I want to be able to just press the language key. Now, this language override is still a good feature, and I could see myself utilising it if I could create additional AZERTY layouts (one for French, one for English, and one for French and English) but then you have the wrong language name on the space bar.

I also want to be able to rename the layouts I'm using so that instead of it saying 'FR - Azerty' on the space bar it would say 'Français'. 'FR - Azerty' is ugly and I spend a lot of time looking at my keyboard. This is important for a nicer implementation of the feature in the previous paragraph, because if I have a double language input, I want to give it a name that means something to me, in my example probably 'Franglais', and I obviously can't have a keyboard that says 'FR - Azerty' but only types in English.

To sum up:
1. I want to be able to create multiple AZERTY keyboards and assign each a different language, rather than being stuck with QWERTY for English; and
2. I want to be able to rename the keyboards, so that I can choose what appears on the space bar.

Perhaps these features are there already but I'm pretty sure that I scoured every nook and cranny of the settings application to no avail.
